Userland/Libraries/LibJS/Runtime/Temporal/Calendar.h

```diff
@@ -83,4 +84,58 @@ ThrowCompletionOr<Object*> default_merge_calendar_fields(VM&, Object const& fiel
 u16 to_iso_day_of_year(i32 year, u8 month, u8 day);
 u8 to_iso_day_of_week(i32 year, u8 month, u8 day);
 
+// https://tc39.es/proposal-temporal/#table-temporal-calendar-methods-record-fields
+struct CalendarMethodsRecord {
```

Comment:
We tend to leave "Record" off the names of structs, unless there's a naming conflict. There doesn't appear to be one, so this would just be `CalendarMethods`.